english/stargazing/stargazing.md

```diff
@@ -0,0 +1,5 @@
+"Look at the stars" she said as she looked at him. They were laying on the lush green Great Lawn of the Central park, she likes admiring the sky because she could imagine themselves in another planet, maybe outside of the galaxy or somewhere out in the cosmos.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.